parquet-cli/src/main/java/org/apache/parquet/cli/commands/ShowFooterCommand.java

```diff
@@ -64,13 +65,19 @@ public int run() throws IOException {
     return 0;
   }
 
+  abstract class MixIn {
+    @JsonIgnore
+    abstract int getInputFile();
```

Comment:
Is there any better way to do this? What about annotating the `getInputFile` at https://github.com/apache/parquet-mr/pull/1242/files#diff-69ea2eba95ed65129e4a4a9e5807807a3c138ea39f2eddf5d6f6b8f2a3b51c73R115?

Reply:
You mean this is a workaround to get rid of the test failure at the cost of a new dependency?

Reply:
Yes

Reply:
I mean the UT will fail if just annotating the `getInputFile` method,  create a MixIn class here (parquet-cli module)  to workaround.

Parquet project already has a dependency of  `jackson-annotations`  library in some other modules. So I don't think   this PR will add a new dependency in parquet-hadoop module.

<img width="678" alt="image" src="https://github.com/apache/parquet-mr/assets/3626747/4fbf536c-29b4-458e-b267-9cf8bdc04d1d">